test/fixtures/inspector-instrumentation-breakpoint/main.js

```diff
@@ -0,0 +1 @@
+require("./dep");
```

Comment:
Nit: We don't lint files in the `fixtures` directory (because many of them have intentional errors, for example), but it would probably be good to put a line break at the end of this file. (This is a nit and you can ignore it if you don't want to be bothered with it.)